networking/ovn_kubernetes_network_provider/configuring-secondary-external-gateway.adoc

```diff
@@ -0,0 +1,44 @@
+:_content-type: ASSEMBLY
+[id="configuring-secondary-external-gateway"]
+= Configure an external gateway through a secondary network interface
+include::_attributes/common-attributes.adoc[]
+:context: configuring-secondary-external-gateway
+
+toc::[]
+
+As a cluster administrator, you can configure an external gateway on a secondary network.
+
+This feature offers the following benefits:
+
+- Granular control over egress traffic on a per-namespace basis
+- Flexible configuration of static and dynamic external gateway IP addresses
+- Support for both IPv4 and IPv6 address families
+
+[id="{context}_prerequisites"]
+== Prerequisites
+
+* Your cluster uses the OVN-Kubernetes network plugin.
+* Your infrastructure is configured to route traffic from the secondary external gateway.
+
+[id="{context}_how-the-configuration-process-works"]
+== How {product-title} determines the external gateway IP address
+
+You configure a secondary external gateway with the `AdminPolicyBasedExternalRoute` custom resource from the the `k8s.ovn.org` API group. The custom resource supports static and dynamic approaches to specifying an external gateway's IP address.
+
+Static assignment:: You specify an IP address directly.
+Dynamic assignment:: You specify an IP address indirectly, with a pod selector and an optional network attachment definition.
```

Comment:
`with a namespace and pod selectors`
The namespace selector is now also mandatory. The reason is to ensure that the CR is only using the gateway pods that reside in a namespace that matches a label criteria. 

Without enforcing the namespace selector, the user could create by mistake a rule that includes pods that are perhaps not meant to be used as gateway pods for a given instance of a AdminPolicyBasedExternalRoute CRD.